apps/omg_watcher/lib/omg_watcher/api/account.ex

```diff
@@ -31,10 +31,16 @@ defmodule OMG.Watcher.API.Account do
 
     # PaymentExitInfo.all_exit_infos() takes a while.
     {:ok, standard_exits} = PaymentExitInfo.all_exit_infos()
-    active_standard_exiting_utxos = OMG.Watcher.ExitProcessor.Core.active_standard_exiting_utxos(standard_exits)
+    {:ok, in_flight_exits} = PaymentExitInfo.all_in_flight_exits_infos()
+
+    active_exiting_utxos =
```

Comment:
consider to left a comment to link the issue for people to understand why filtering both SE and IFE inputs: `https://github.com/omgnetwork/private-issues/issues/41`